### DISCUSSION TITLE
Little timer when puppets make first connection to irc server when start

### DISCUSSION THREAD

**Discussion by BCC-BGChatCommunity** [qaisjp/go-discord-irc#114] [Ideas]
Hello. I have some problem with irc side. When i restart discord bot and then start it with many users inside irc have limits for connections and after many they kill all others and make them died (and can't join again). I can make except my ip but if you have some small timer for each puppet when join them on start will be great. 
Thanks

**Comment by qaisjp** [comment]
You will absolutely need custom configuration on the IRC side if you have lots of Discord users online, as afaik there's usually a limited number of connections that can originate from a single IP address.

I'm not sure what the benefit tradeoff of having bots take a full minute to join (because of a delay between each puppet) vs. someone _also_ configuring their IRC server to allow puppets to bypass other restrictions.

If you _really_ need this, I can add it, but it's probably easier and quicker (for bots to rejoin) for you to add additional configuration on your side!
